src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -894,30 +900,53 @@ internal virtual void PrepareSession()
                 // Store the UserAgent string
                 WebSession.UserAgent = UserAgent;
             }
-
-            if (Proxy is not null)
+            // Proxy and NoProxy parameters are mutually exclusive
+            // If NoProxy is provided, WebSession will turn off the proxy
+            // and if Proxy is provided NoProxy will be turned off
+            if (NoProxy.IsPresent)
```

Comment:
We have code conventions. Please look docs and existing code. Also please look CI CodeFactor.

Here please and new line after `}` and before the comment.